onboarding_tool/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+project(onboarding_tool)
+
+if(UNIX)
+    add_executable(onboarding_tool
+        ${PROJECT_SOURCE_DIR}/obtmain.c
+        ${iotivity-lite_SOURCE_DIR}/api/oc_introspection.c
+    )
+
+    target_link_libraries(onboarding_tool
+        iotivity-client
+    )
+
+    file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/onboarding_tool_creds)
```

Comment:
It seems odd that this is required everywhere, does the application not make its own directories?

Reply:
That seems very odd, as you then can't run the application from a different folder or move it, but I guess this solves the problem for now 👍